Examples/BinaryBH/Main_BinaryBH.cpp

```diff
@@ -57,6 +59,13 @@ int runGRChombo(int argc, char *argv[])
 
     std::chrono::time_point<Clock> start_time = Clock::now();
 
+    // Add a scheduler to GRAMR to calls specificPostTimeStep on every AMRLevel
+    // at time 0
+    CallFunctionOverLevelsPtr<> call_now(
+        [](GRAMRLevel *level) { level->specificPostTimeStep(); });
+    call_now.execute(bh_amr); // call 'now' really now
+    // bh_amr.schedule(call_now); // or call 'now', after 1st plot level is done
```

Comment:
I like having the commented out example for reference, but then I think it would be helpful to actually define an a_interval and pass it here to show how that works. Then the comment can say that the task is performed post plotlevel, at every a_interval (and in reverse order). This will make it easier for people to copy without reading the scheduler files!